corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
i'm pretty confused. you'd think `tempfile.gettempdir()` would work if `/tmp` was writeable. (the `/home/travis/tmp` doesn't surprise me since we never created that dir)